### PR TITLE
Made some updates to reflect status as of Android 12.

### PR DESCRIPTION
**Pull request by pratyakshm**
Made values under Android version section consistent.

Added new rule: Under the Android versions section, input the name of the latest Android version first, followed by older Android version(s), and separated by "&" (ampersand symbol). For instance: 12 & 11.

Removed "Some URIs are not supported" text, since the example that was provided: ```wsa://com.android.settings``` correctly opens the Android Settings app on Android 12 based WSA (version: 2204.40000.20.0).

### CODE REVIEW

File: README.md
Comment (riverar) [inline README.md:57]: What's the thinking behind this change? This URI still works.

File: README.md
Comment (riverar) [inline README.md:122]: Think we should update all these using the rules I proposed above.

**Review by riverar — CHANGES_REQUESTED** [review]

File: README.md
Comment (pratyakshm) [reply README.md:57]: In master branch, the note mentions:
Some URIs are not supported in WSA, such as:
```
wsa://com.android.settings
```
However, this URI is supported in Android 12 version of WSA, so running the URI from Run dialog opens Android Settings app.

File: README.md
Comment (pratyakshm) [reply README.md]: Done!

**Comment by pratyakshm** [comment]
@riverar All done, waiting for you to review before I can merge this PR.

**Pull request merged**
